crates/core_arch/src/aarch64/neon/mod.rs

```diff
@@ -971,6 +971,40 @@ pub unsafe fn vaddvq_u64(a: uint64x2_t) -> u64 {
     vaddvq_u64_(a)
 }
 
+/// Vector multiply accumulate
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(fmla))]
+pub unsafe fn vmla_f32(a: float32x2_t, b: float32x2_t, c: float32x2_t) -> float32x2_t {
+    simd_fma(b, c, a)
+}
+
+/* FIXME: simd_fma doesn't support float64x1_t`
+/// Vector multiply accumulate
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(fmla))]
+pub unsafe fn vmla_f64(a: float64x1_t, b: float64x1_t, c: float64x1_t) -> float64x1_t {
+    simd_fma(b, c, a)
```

Comment:
You can use the `mul_add` method on `f64`.

Reply:
Sorry for delay an thank you for review. I will update this PR.